packages/core/scripts/collateral-umip/3_Verify.js

```diff
@@ -2,27 +2,60 @@
 // It can be run on mainnet after the upgrade is completed or on the local Ganache mainnet fork to validate the
 // execution of the previous two scripts. This script does not need any wallets unlocked and does not make any on-chain
 // state changes. It can be run as:
-// yarn truffle exec ./scripts/collateral-umip/3_Verify.js --network mainnet-fork --collateral 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 --fee 0.1
+// yarn truffle exec ./scripts/collateral-umip/3_Verify.js --network mainnet-fork --collateral 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 --fee 0.1 --collateral 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48 --fee 400
 
 const assert = require("assert").strict;
 
 const AddressWhitelist = artifacts.require("AddressWhitelist");
 const Store = artifacts.require("Store");
+const ERC20 = artifacts.require("ERC20");
+
+const { parseUnits } = require("@ethersproject/units");
+const _ = require("lodash");
 
 const argv = require("minimist")(process.argv.slice(), { string: ["collateral", "fee"] });
 
+async function getDecimals(collateralAddress, decimalsArg) {
```

Comment:
not really a suggestion for this PR but we probably could use some organizational thing where we can avoid code duplication. i usually end up throwing stuff like this in a utils.js file. not a blocker by curious on your thoughts here.

Reply:
yeah, this small scale duplication is difficult to prevent since it would, in this case require adding an extra file to hold a function without a lot of context. Adding a utils.js file for good measure :).

Reply:
Done.